addon/components/resize-sensor.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@ember/component';
+import { next } from "@ember/runloop"
+
+import ResizeSensor from 'resize-sensor';
+
+export default Component.extend({
+  classNames: ['resize-sensor-container'],
+
+  didInsertElement() {
+    const { element } = this;
+    this._resizeSensor = new ResizeSensor(element, () => {
+      next(this, () => {
```

Comment:
Sorry, one last change here:

```js
this._resizeSensor = new ResizeSensor(element, () => {
  if (this._nextSendAction !== null) {
    return;
  }

  this._nextSendAction = next(() => {
    this._nextSendAction = null;
    
    const onResize = this.get('onResize');
    
    if (typeof onResize === 'function') {
      onResize(element);
    } else {
      this.sendAction('onResize', element);
    }
  })
});
```

You'll also need to add `_nextSendAction` to the class itself and set its default to null. This will prevent the resize sensor from scheduling the action being sent more than once

Reply:
What is _nextSendAction? I can't find it on google.